documentation-generator/src/test/java/tech/picnic/errorprone/documentation/BugPatternTaskListenerTest.java

```diff
@@ -0,0 +1,90 @@
+package tech.picnic.errorprone.documentation;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.picnic.errorprone.documentation.DocumentationGenerator.DOCS_DIRECTORY;
+
+import com.google.common.io.Resources;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+final class BugPatternTaskListenerTest extends TaskListenerCompilerBasedTest {
```

Comment:
These tests should arguably be part of `BugPatternExtractorTest`, or `DocumentationGeneratorTaskListenerTest`. (I guess the former, since otherwise the latter will be very bulky.)

Reply:
Deal, makes sense.